### ISSUE TITLE
Stopping on server with no endpoints cause NPE

### ISSUE THREAD

**Issue by rockem** [rockem/blink-java#17]
When just instantiating blink and then stopping it, it will cause NPE

**Comment by the123saurav** [comment]
Are you taking contributions?

**Comment by rockem** [comment]
yup, although since jcenter is closed now, in order to republish this library we need to move it to maven first
